### ISSUE TITLE
Fix pyflakes for names

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6990]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6990|
|Type|enhancement|
|Created|2014-02-27 12:11:54Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-names-6990|

Attachments:

* [6990-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f44/f44e20e17b7f7a559cb16539ee1f54ded70f5961/c749fe323bb7a055092e35f4059787c340a3c6b8.diff) (1985 bytes) - added by adiroiban on 2014-02-27 12:16:59Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6990 6990
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_names_6990 branches/pyflakes-names-6990
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__names names
keywords__ 
time__1393503114000000 1393503114000000
changetime__1393624706000000 1393624706000000
version__None None
owner__rwall rwall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

warnings were not used and I did a search and did not find any other places importing warnings from names.root. Even if they do, I think this is a bad practice.

I have renamed 'msg' variables to not overwrite the imported msg module. I think that 'message' is a better name for this variable... also did a wrap for long lines.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_2"></a>|@wallrj commented|
|-|-|

(In [41776]) Branching to 'pyflakes-names-6990'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj commented|
|-|-|

(In [41777]) Apply 6990-1.diff from adiroban. Refs [#6990](https://github.com/twisted/twisted/issues/6990)

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_5"></a>|@wallrj set owner to @adiroiban<br>@wallrj set status to `new`|
|-|-|

Thanks Adi,

The patch looks good.

I ran pyflakes over twisted/names and docs/projects/names and there were no warnings.

Also confirmed that all twisted.names tests still pass.

I added a misc newsfile.

So I'll go ahead and merge.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_7"></a>|@wallrj set status to `closed`|
|-|-|

(In [41779]) Merge pyflakes-pair-6993

Author: adiroiban
Reviewer: rwall
Fixes: [#6990](https://github.com/twisted/twisted/issues/6990)

Remove unused imports from modules in Twisted Names. Twisted Names is now
"pyflakes clean".
